sdk/core/http/.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
```

Comment:
At part of a clean-up effort this file should be deleted. The gitignore should also be merged in with the root and removed.

Reply:
Will do!